permits/forms.py

```diff
@@ -1090,8 +1104,10 @@ def __init__(self, *args, **kwargs):
         if services.GeoTimeInfo.DATE not in required_info:
             del self.fields["starts_at"]
             del self.fields["ends_at"]
-
-        if services.GeoTimeInfo.GEOMETRY not in required_info:
+        if (
```

Comment:
`if((services.GeoTimeInfo.GEOMETRY and services.GeoTimeInfo.GEOCODED_GEOMETRY) not in required_info)`

Reply:
Nope